test/e2e/play_kube_test.go

```diff
@@ -4931,4 +4944,24 @@ spec:
 		Expect(strings.Count(kube.OutputToString(), "Pod:")).To(Equal(1))
 		Expect(strings.Count(kube.OutputToString(), "Container:")).To(Equal(1))
 	})
+
+	It("podman play kube test with hostPID", func() {
+		err := writeYaml(podWithHostPIDDefined, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube).Should(Exit(0))
+
+		logs := podmanTest.Podman([]string{"pod", "logs", "-c", "test-hostpid-alpine", "test-hostpid"})
+		logs.WaitWithDefaultTimeout()
+		Expect(logs).Should(Exit(0))
+		Expect(logs.OutputToString()).To(Not(Equal("1")))
```

Comment:
```suggestion
		Expect(logs.OutputToString()).To(Not(Equal("1")), "PID should never be 1 because of host pidns")
```
because  you have to repush anyway please add a description here so log readers can make sense of this error when it ever fails

Reply:
Right, that makes sense. Thanks for the suggestion